Updating the helper function for goDebug to change the focus to the c…

GregGoldman opened the pull request:
…ommand window instead of the editor window. Change is only one line in the file.

GregGoldman committed:
Updating the helper function for goDebug to change the focus to the command window instead of the editor window. Change is only one line in the file.

JimHokanson commented:
Hi Greg,

I just took a closer look at this. It looks technically correct, but there are three things I'd change.

GregGoldman committed:
Work for goDebug functionality
Wrapped matlab's openAndGoToLine with our own
moved a bunch of files around to streamline

GregGoldman commented:
Main odd thing is trying to put goDebugHelper.m in the private folder makes it invisible to the mex function. Not sure why this is happening because the private folder should be visible to functions in the parent folder.
